### PR TITLE
11.0 mig mail footer notified partner

### PR DESCRIPTION
**Pull request by cvinh**

### CODE REVIEW

**Comment by cvinh** [comment]
I believe the notification must be at the beginning of the body to be read.
We had a story here : somebody has been fired because he answered bad words in reply and his manager was in the followers...

**Comment by hbrunn** [comment]
please increase code coverage and fix the warnings from https://travis-ci.org/OCA/social/jobs/398709994#L598

**Comment by eLBati** [comment]
@cvinh do you plan to work on this?

**Comment by cvinh** [comment]
I'm sorry I don't understand how to increase codecov ... please help..?

File: mail_footer_notified_partner/models/mail_followers.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+from odoo import models, api
+from odoo.tools.translate import _
+
+
+class MailNotification(models.Model):
+    _inherit = 'res.partner'
+
+    @api.multi
+    def _notify_by_email(
```
Comment (Tonow-c2c) [inline mail_footer_notified_partner/models/mail_followers.py]: this method is not any-more available in V 11.0
https://github.com/odoo/odoo/blob/10.0/addons/mail/models/res_partner.py#L171
not in https://github.com/odoo/odoo/blob/11.0/addons/mail/models/res_partner.py

**Review by Tonow-c2c — CHANGES_REQUESTED** [review]

**Comment by Tonow-c2c** [comment]
@cvinh you can add may be this new test in you want to increase codecov https://github.com/OCA/social/pull/448/files#diff-76f93c4b8d73d0e1bcc1cc16fcd920dcR29-R31
:wink:

**Comment by cvinh** [comment]
@Tonow-c2c thank you

**Comment by rafaelbn** [comment]
@cvinh you are free to reopen this one if you need it

**Comment by cvinh** [comment]
@rafaelbn v11 is deprecated for us but I opened the same on v14 here https://github.com/OCA/social/pull/857

**Pull request closed without merging**